cmd/drone-server/inject_login.go

```diff
@@ -172,5 +172,13 @@ func provideRefresher(config config.Config) *oauth2.Refresher {
 			Source:       oauth2.ContextTokenSource(),
 		}
 	}
+	if config.Gitea.ClientID != "" {
```

Comment:
Maybe add a switch instead of another if?

Reply:
Updated